src/script/event/EventService.ts

```diff
@@ -153,41 +155,6 @@ export class EventService {
     }
   }
 
-  /**
-   * Load a replacing event from database.
-   * To be used in certain cases (during some cases with quoting) when the original message cannot be found.
-   *
-   * @param conversationId ID of conversation
-   * @param eventId ID of event to retrieve
-   */
-  async loadReplacingEvent(conversationId: string, eventId: string): Promise<EventRecord> {
-    if (!conversationId || !eventId) {
-      this.logger.error(`Cannot get event '${eventId}' in conversation '${conversationId}' without IDs`);
-      throw new ConversationError(BASE_ERROR_TYPE.MISSING_PARAMETER, BaseError.MESSAGE.MISSING_PARAMETER);
-    }
-
-    try {
-      if (this.storageService.db) {
-        return await this.storageService.db
-          .table(StorageSchemata.OBJECT_STORE.EVENTS)
-          .where('id')
-          .equals(eventId)
-          .filter(record => record.conversation === conversationId)
-          .first();
-      }
-
-      const records = (await this.storageService.getAll(StorageSchemata.OBJECT_STORE.EVENTS)) as EventRecord[];
-      return records
-        .filter(record => record.conversation === conversationId && record.data?.replacing_message_id === eventId)
-        .sort(compareEventsById)
-        .shift();
-    } catch (error) {
-      const logMessage = `Failed to get event '${eventId}' for conversation '${conversationId}': ${error.message}`;
-      this.logger.error(logMessage, error);
-      throw error;
-    }
```

Comment:
This was actually not needed (as the `loadEvent` was already checking for replacing_message_id under the hood)